includes/addons.php

```diff
@@ -3,19 +3,6 @@
  * Some of the code in this library was borrowed from the TGM Updater class by Thomas Griffin. (https://github.com/thomasgriffin/TGM-Updater)
  */
 
-/**
- * Setup plugins api filters
- *
- * @since 1.8.5
- */
-function pmpro_setupAddonUpdateInfo() {
-	add_filter( 'plugins_api', 'pmpro_plugins_api', 10, 3 );
-	add_filter( 'pre_set_site_transient_update_plugins', 'pmpro_update_plugins_filter' );
-	add_filter( 'http_request_args', 'pmpro_http_request_args_for_addons', 10, 2 );
-	add_action( 'update_option_pmpro_license_key', 'pmpro_reset_update_plugins_cache', 10, 2 );
```

Comment:
We need to keep the pmpro_reset_update_plugins_cache hook/callback in PMPro core. So add this section back and then remove the first 3 add_filters.